src/driver.ts

```diff
@@ -213,8 +213,9 @@ export abstract class CMakeDriver implements vscode.Disposable {
 
   getEffectiveSubprocessEnvironment(opts?: proc.ExecutionOptions): proc.EnvironmentVariables {
     const cur_env = process.env as proc.EnvironmentVariables;
+    const kit_env = (this.ws.config.ignoreVsEnv) ? {} : this.getKitEnvironmentVariablesObject();
```

Comment:
I think it would be better to check for `ignoreVsEnv` here:
https://github.com/vector-of-bool/vscode-cmake-tools/blob/b3a7aca1f86052ef94111163514906474c3d1f47/src/kit.ts#L638-L648
Or maybe rename the option to `ignoreKitEnv`?

Reply:
Renamed per your suggestion!